src/components/SwiperFlatList/index.js

```diff
@@ -167,14 +167,14 @@ export default class SwiperFlatList extends PureComponent {
       ref: component => {
         this.flatListRef = component;
       },
+      ...props,
       keyExtractor: this._keyExtractor,
       horizontal: !vertical,
       showsHorizontalScrollIndicator: false,
       showsVerticalScrollIndicator: false,
       pagingEnabled: true,
       onMomentumScrollEnd: this._onMomentumScrollEnd,
```

Comment:
Could you move only `onMomentumScrollEnd: this._onMomentumScrollEnd` after `..props`.

```
onScrollToIndexFailed: this._onScrollToIndexFailed,
...props,
onMomentumScrollEnd: this._onMomentumScrollEnd,
```